pyanaconda/ui/tui/spokes/storage.py

```diff
@@ -576,3 +596,256 @@ def input(self, args, key):
+            # TRANSLATORS: 's' to rescan devices
+            if key.lower() == C_('TUI|Spoke Navigation|Partitioning', 's'):
+                text = _("Warning: This will revert all changes done so far.\nDo you want to proceed?\n")
+                question_window = YesNoDialog(text)
+                ScreenHandler.push_screen_modal(question_window)
+                if question_window.answer:
+                    print(_("Scanning disks. This may take a moment..."))
+                    storage_initialize(self.storage, self.data, self.storage.devicetree.protected_dev_names)
+                    self.data.mount.clear_mount_data()
+                    self._gather_mount_data_info()
+                self.redraw()
+                return InputState.PROCESSED
+            # TRANSLATORS: 'c' to continue
+            elif key.lower() == C_('TUI|Spoke Navigation', 'c'):
+                self.apply()
+
+            return super(MountPointAssignSpoke, self).input(args, key)
+
+        return InputState.PROCESSED
+
+    def apply(self):
+        """ Apply our selections. """
+        for md in self._mds.values():
+            if md.modified and (md.reformat or md.mount_point):
+                self.data.mount.add_mount_data(md)
+
+class ConfigureDeviceSpoke(NormalTUISpoke):
+    """ Assign mount point to a block device and (optionally) reformat it """
+    category = SystemCategory
+
+    def __init__(self, data, storage, payload, instclass, mount_data):
+        NormalTUISpoke.__init__(self, data, storage, payload, instclass)
+        self._container = None
+        self._md = mount_data
+        self.title = N_("Configure device: %s") % mount_data.device
+
+    @property
+    def indirect(self):
+        return True
+
+    def refresh(self, args=None):
+        NormalTUISpoke.refresh(self, args)
+
+        self._container = ListColumnContainer(1)
+
+        fmt = get_format(self._md.format)
+        if fmt and fmt.mountable:
+            self._container.add(TextWidget("Mount point: %s" % self._md.mount_point or "none"), self._assign_mount_point, self._md)
+        else:
+            self._container.add(TextWidget("Mount point: [%s]" % fmt.name), lambda x: self.redraw(), None)
+        self._container.add(TextWidget("Format: %s" % self._md.format or "none"), self._set_format, self._md)
+        if ((self._md.orig_format and self._md.orig_format != self._md.format) or self._md.mount_point == "/"):
+            # changing format implies reformat and so does "/" mount point
+            self._container.add(CheckboxWidget(title="Reformat", completed=self._md.reformat), lambda x: self.redraw(), None)
+        else:
+            self._container.add(CheckboxWidget(title="Reformat", completed=self._md.reformat), self._switch_reformat, self._md)
+
+        self.window.add_with_separator(self._container)
+
+        message = _("Choose from above to assign mount point and/or set format.")
+        self.window.add_with_separator(TextWidget(message))
```

Comment:
I guess you can inline the `message` here. You have bigger lines above which are ok.